examples/features/multiplex/README.md

```diff
@@ -0,0 +1,11 @@
+# Multiplex
+
+A `ClientConn` can be shared by two stubs and two servicers can share a server.
```

Comment:
"...can share a `Server`."

Or `grpc.ClientConn` and `grpc.Server`?

Reply:
Done